Alert/doc/spec.md

```diff
@@ -0,0 +1,148 @@
+# Alert data model
+
+## Description
+This entity models an alert generated either by a user or by a device. This models could be used to send alerts related to traffic jam, accidents, weather conditions, high level of pollutants and so on.
+The purpose of the model is to support the generation of notifications for a user or trigger other actions,
+based on such alerts.
+
+### Alert definition
+Alerts are context data generated by final users (humans) or devices, such as smartphones, service, or sensors.
+
+An alert is generated by a specific situation. The main features of an alert is that it is not predictable and it is not a recurrent data. That means that an alert could be an accident or a high level of pollutants measure, additionally it could be the fall down of a patient or a car driving in the opposite direction.
+
+The model has been defined to support different scenarios inside [SmartSDK](https://smartsdk.eu) project:
```

Comment:
I would not reference any project here, although you can add a sentence in the README or at the end of the data model description.